package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-web",
   "description": "Digital service mock to claim public money in the event property subsides into mine shaft.",
-  "version": "2.3.0",
+  "version": "3.0.0",
```

Comment:
Are the changes backwards compatibile? i.e. if you don't need to make changes to add a password for running in AWS then this might be a minor upgrade than a major one.

Reply:
I had originally set it as a minor upgrade but went to major as 1. setting `tls: value.isProd ? {} : undefined` in the `catboxOptions` will break the AWS deploy and 2. it seemed a clean break to move all Azure specific code to `3.0.0` and above leaving the `2.x.x` branch known to work in AWS (I am assuming all merges from now will be for Azure)